doc/cephfs/fs-volumes.rst

```diff
@@ -790,32 +790,38 @@ pinned to one of the available ranks on the file system.
 Subvolume quiesce
 -----------------
 
-It may be needed to pause IO to a set of subvolumes of a given volume (file system).
-A good example of such case is a consistent snapshot spanning multiple subvolumes.
-The task arises often in an orchestrated environment such as Kubernetes, where a single deployed application
-can work with many mounted subvolumes across several hosts. When a snapshot of such a system is needed,
-the application may not find the result consistent unless the snapshots were taken
-during an active write pause.
+Distributed applications that run against different mount points of the same Ceph file system
+are facing a challenge of consistent snapshots for backups and disaster recovery. The challenge is due to
+the fact that CephFS snapshots dont provide strong consistency guarantees when there are multiple
+clients performing writes. Even when the application is using file system flushes to synchronize
```

Comment:
s/Distributed applications that run against different mount points of the same Ceph file system
are facing a challenge of consistent snapshots for backups and disaster recovery. The challenge is due to
the fact that CephFS snapshots dont provide strong consistency guarantees when there are multiple
clients performing writes. Even when the application is using file system flushes to synchronize checkpoints across its distributed components, there aren't guarantees that all acked writes will be part of a given snapshot./Because CephFS snapshots do not provide strong-consistency guarantees in cases involving writes performed by multiple clients, distributed applications running against different mount points of the same Ceph file system face a serious challenge. Even in a case where an application uses file system flushes to synchronize checkpoints across its distributed components, there is no guarantee that all acknowledged writes will be part of a given snapshot./

Reply:
This is a tricky one. I took the liberty to edit your variant, please review the new version. It's funny that you edited out the very reason we opened this PR: an explicit reference to backups and disaster recovery.

I'm adding it back because I like to stick to the plan 😅

Also, I'm surprised you suggest beginning a section with `Because`. I'm not qualified to argue, but I would like to double-check, and I do it boldly by removing the word in my edit - subject to your review.

Reply:
The problem here is that "are facing a challenge of consistent snapshots for backups and disaster recovery" just isn't meaningful.

Here are some things that might be intended, and that make more sense in English:

(1) Distributed applications that run against different mountpoints of the same Ceph file system have difficulty creating consistent snapshots for backups and for disaster recovery.

(2) When distributed applications run against different mountpoints of the same Ceph file system, those applications have trouble creating consistent snapshots for backups and for disaster recovery.

The next sentence also uses the word "challenge" in a way that distinguished-level English just doesn't permit. I might suggest this:

For:

> The challenge is due to the fact that CephFS snapshots dont provide strong consistency guarantees when there are multiple clients performing writes.

I suggest:

> This is because CephFS snapshots don't provide strong-consistency guarantees when multiple clients are performing writes.

If backups and disaster recovery are the real star of the show, then I might suggest putting them up front in the first sentence of the section:

> When preparing backups and preparing for disaster recovery, there are some important challenges to bear in mind if you use  distributed applications that run against different mount points of the same Ceph file system.

In this context, you can use the word "challenges" and it doesn't sound wrong.